src/utils/build-url-parameters.js

```diff
@@ -0,0 +1,72 @@
+import { FORMAT, RESIZE, CROP, QUALITY, BACKGROUND_COLOR } from '../constants';
+import { parametersMapper, valuesMapper } from './mappers';
+
+const getParameterPrefix = (currentUrlParameters) => !currentUrlParameters ? '?' : '&';
+
+const buildUrlParametersFormat = ({ key, value, currentUrlParameters, includeMimeType }) => {
```

Comment:
This is just personal preference, but I would prefer an approach where the `currentUrlParameters` didn't have to be passed to the `buildUrlParameters` function's.

Something like gathering all the resulting unprefixed strings in an array, calling `array.join('&')` and prefixing the resulting string with `?`. This way the parameter builders wouldn't have to know what has been built so far.

Again, this is just preference, I don't mind the way it is now.

Reply:
Actually, I think it makes a lot of sense. I refactored my code to be in line with your suggestion. Thanks 👌